galaxy_ng/app/constants.py

```diff
@@ -182,18 +200,36 @@ class DeploymentMode(enum.Enum):
         ),
         "ui_category": _("Ansible Repository"),
     },
+    "container.view_containernamespace": {
```

Comment:
A lot of the permissions you've added here don't actually do anything. Namespaces, for example, are viewable to everyone who is authenticated. We don't have the notion of a private namespace. I'd like to keep this file limited to the set of permissions that are actually enforced.

I think these will start to make sense once we fully integrate organizations, but for now this will just make the UI confusing.

Reply:
I need a discrete list of what's needed in that case. I had to write a script to find the missing "read" perms for each category because there's no pointers otherwise.

Reply:
This file is the discrete list of what's needed. That's why I sent it to you. These are the permissions we care about.